src/builders/ModelBuilder.ts

```diff
@@ -45,6 +45,10 @@ export class ModelBuilder extends Builder {
                 ...col.dataType && { type: col.dataType },
                 ...col.comment && { comment: col.comment },
                 ...col.defaultValue && { defaultValue: col.defaultValue },
+                ...col.onUpdate && { onUpdate: col.onUpdate },
+                ...col.onDelete && { onDelete: col.onDelete },
+                // @TODO fix this by creating a typescript definition for model
+                // ...col.references && { references: col.references as ModelAttributeColumnReferencesOptions },
```

Comment:
Probably the following type must be imported?

```ts
import { ModelAttributeColumnOptions, ModelAttributeColumnReferencesOptions } from 'sequelize';
```

Reply:
This is not about importing types. This is about reading references directly from postgres tables instead of taking it from file definition -  this was just a try